### DISCUSSION TITLE
ASUS CETRA Supernova

### DISCUSSION THREAD

**Discussion by EoKc** [seerge/g-helper#2786] [Ideas]
Hello,

Is there any plan for supporting asus cetra supernova? I am using AC phone application right now for the changing the active noise cancellation. Thank you for the G-HELPER app it is life saver btw :)

**Comment by seerge** [comment]
@EoKc most probably not :) As far as I remember (from experience with other asus headphones) PC version of Armoury Crate also wasn't offering any support of it, only mobile - the one you use now already.

**Reply by EoKc** [reply]
Thank you for reply @seerge , but this one connecting via type c dongle maybe it will change something

**Reply by seerge** [reply]
Did you check if laptop version of AC offers any options for it ?

**Reply by EoKc** [reply]
Honestly i did not but i will right now i hope it is not messed up with G-HELPER

**Reply by EoKc** [reply]
![resim_2024-06-27_190611836](https://github.com/seerge/g-helper/assets/30772731/b333c745-6985-4da9-afb7-c434a1cd5e18)
Their software did not work properly. 

![resim_2024-06-27_190654730](https://github.com/seerge/g-helper/assets/30772731/bd7b7dc4-b54d-4b32-829c-b48660334a0e)

I can only see battery status on icon tab 

![resim_2024-06-27_190907858](https://github.com/seerge/g-helper/assets/30772731/8aacd326-d7f9-46b5-9600-fa12ea270667)

Also there is a tab on aursa sync page too

**Reply by EoKc** [reply]
![resim_2024-06-27_191704160](https://github.com/seerge/g-helper/assets/30772731/8c97f27e-9dd4-473e-a092-21bb09203bd5)

I found youtube video. In video there is a settings for ANC on/off

**Reply by EoKc** [reply]
![resim_2024-06-27_191940146](https://github.com/seerge/g-helper/assets/30772731/c78ec437-c014-4fbc-a696-2143d606c9cc)

![resim_2024-06-27_192003335](https://github.com/seerge/g-helper/assets/30772731/748edacd-5994-4f33-a844-e8478d16308a)

![resim_2024-06-27_192015945](https://github.com/seerge/g-helper/assets/30772731/e8b9a514-653d-4be0-a7cc-641dc07fa1c3)

**Reply by seerge** [reply]
So in your case AC doesn't offer much, but in someone's youtube you have found that it does?

**Reply by EoKc** [reply]
My AC is not working properly i don't know why but on some review video people using AC for the changing the ANC settings, power settings and lightening. like a headset

**Reply by IceStormNG** [reply]
I had some of those here (not the supernova, but the regular ones) and the settings are not stored on the headphones, but handled via a driver from AC on your PC. 
Especially all the audio stuff. GHelper does no real-time processing, but only offers the ability to change the on-board memory profiles, which those headphones do not offer.
